### PR TITLE
Convert theme switcher into button

### PR DESCRIPTION
**Pull request by GuillemPM**
- **Yes, I just changed the theme switcher input for a button.**
- **Yes, I invested 5 hours on my life on this.**
- **Yes, I had to recreate all the styles.**

# Why?
**_Because this thing gave me anxiety_**
### Before (the input disappears because it needs to wait the state to retrieve the theme and so it can be checked)
input-before

### AFTER (it doesn't disappear because I'm a master pro 3000)
after-themeswitcher

# DON'T ASK ME WHY, JUST MERGE THIS 😭

### CODE REVIEW

File: frontend/src/components/router-head/theme-script.tsx [hunk 1]
```diff
@@ -8,6 +8,10 @@ export const ThemeScript = () => {
             .setAttribute('data-theme',
                 localStorage.getItem('${themeStorageKey}') ??
                 (window.matchMedia('(prefers-color-scheme: dark)').matches ? '${DARK_THEME}' : '${LIGHT_THEME}')
-            )`;
+            );
+        if (document.firstElementChild
+            .getAttribute('data-theme') === '${DARK_THEME}')
+                document.firstElementChild.classList.add('dark');
+        `;
```
Comment (origranot) [inline frontend/src/components/router-head/theme-script.tsx:15]: You don't havve to check if the theme is dark twice.
Maybe we should store the theme in const

**Review by origranot — COMMENTED** [review]
This looks good 🔥 

Well, you are crazy to work on it for 5 hours 👑 

Good job!

File: frontend/src/components/router-head/theme-script.tsx (hunk 1 above)
Comment (GuillemPM) [reply frontend/src/components/router-head/theme-script.tsx:15]: This property allows tailwind to load classes using the format dark:classname, and data-scheme is used to overwrite css variable values, that's why I had to add it

Comment (GuillemPM) [reply frontend/src/components/router-head/theme-script.tsx:15]: Maybe we can optimize this in the future, but I think it's not the big difference right now

**Pull request merged**